amazon/ion/core.py

```diff
@@ -630,6 +633,15 @@ def __iter__(self):
         for key in six.iterkeys(self.__store):
             yield key
 
+    if _PY_VERSION >= 3:
+        def __str__(self):
+            str_repr = '{'
+            for key in six.iterkeys(self.__store):
+                str_repr += '\'%s\': %s, ' % (key, self.__store[key][0].__repr__())
```

Comment:
Call `repr(self.__store[key][0])` instead of `self.__store[key][0].__repr__()`. Or, should this be `str(self.__store[key][0])`?